src/constants.ts

```diff
@@ -28,3 +28,4 @@ export const CONST_URL_PRIVACY_POLICY =
   "https://www.mozilla.org/privacy/subscription-services/";
 export const CONST_URL_MONITOR_GITHUB =
   "https://github.com/mozilla/blurts-server";
+export const CONST_DAY_MILLISECONDS = 24 * 60 * 60 * 1000;
```

Comment:
Just because I am both curious (and annoying) and found this interesting. Not sure if any of these would work with this new constant. Maybe only the first one. The second one isn't multiplying by 24, and the others multiply by `100` and not `1000`, so would be off by 10&times;.
```js
git grep -IPin "\b60\s?\*\s?60\b" -- ":(exclude)*.ftl" ":(exclude)*.json" ":(exclude)*.svg" ":(exclude)*.md" | cat

scripts/breach-stats.js:7:  const cd = 24 * 60 * 60 * 1000
scripts/breach-stats.js:8:  const ch = 60 * 60 * 1000
src/app/hooks/useLocalDismissal.test.tsx:74:      { maxAge: 100 * 365 * 24 * 60 * 60 },
src/app/hooks/useLocalDismissal.test.tsx:120:      { maxAge: 100 * 365 * 24 * 60 * 60 },
src/app/hooks/useLocalDismissal.ts:50:          100 * 365 * 24 * 60 * 60;
src/constants.ts:31:export const CONST_DAY_MILLISECONDS = 24 * 60 * 60 * 1000;
```

Reply:
I’m always impressed (a not annoyed) by how you know your way around `grep`. I’ve stumbled across these values as well and decided to stick to replacing the matching values that don’t need additional refactoring.